### PR TITLE
hint fallback to new action if there is no hint for edit action

### PR DESCRIPTION
**Pull request by sivagollapalli**
#1344

### CODE REVIEW

**Comment by sivagollapalli** [comment]
@lucasmazza Could review this PR?

File: Gemfile.lock [hunk 1]
```diff
@@ -294,4 +294,4 @@ DEPENDENCIES
   simple_form!
 
 BUNDLED WITH
-   1.12.5
+   1.13.2
```
Comment (oliverklee) [inline Gemfile.lock]: This change is not related to this PR, I guess. Could you please remove it?

**Comment by sivagollapalli** [comment]
Done the changes. Please let me know if any changes are needed.

**Comment by feliperenan** [comment]
Hi @sivagollapalli thanks for the pull-request and sorry about the delay response 😢 

I'm not familiar with this code at Simple Form. Could you explain how could I test this fix? would be nice if you provide a sample application that I can run this code against.

Thank you

**Comment by feliperenan** [comment]
I'm gonna close this PR since it hasn't recently activity. Fell free to reopen case you are still planing to work on it.
